packages/playwright-core/src/server/har/harTracer.ts

```diff
@@ -460,6 +465,16 @@ export class HarTracer {
     }));
   }
 
+  private _computeHarEntryTotalTime(harEntry: har.Entry) {
+    harEntry.time = [
+      harEntry.timings.dns,
+      harEntry.timings.connect,
+      harEntry.timings.ssl,
+      harEntry.timings.wait,
+      harEntry.timings.receive
+    ].reduce((pre, cur) => (cur || -1) > 0 ? cur! + pre! : pre, 0)!;
```

Comment:
Can any of times be zero? Perhaps `??` is better in this case?

```suggestion
    ].reduce((pre, cur) => (cur ?? -1) > 0 ? cur! + pre! : pre, 0)!;
```